inetum-realjobs-frontend/src/app/components/user-details/user-details.component.html

```diff
@@ -1 +1,54 @@
-<p>user-details works!</p>
+<p-message severity="error" [text]="errorMessage" *ngIf="error" styleClass="p-mr-2"></p-message>
+
+<div class="center-div" *ngIf="loading">
+  <p-progressSpinner></p-progressSpinner>
+</div>
+
+<div class="p-fluid" *ngIf="(!loading && !error)">
+  <p-fieldset >
+    <ng-template  pTemplate="header">Personal Information</ng-template>
+    <form (ngSubmit)="save()" [formGroup]="profileForm" class="gridForm">
+      <div class="p-inputgroup mb-2">
+        <span class="p-inputgroup-addon"><i class="pi pi-user"></i></span>
```

Comment:
Why is this a separate span? Please check [https://www.primefaces.org/primeng/showcase/#/inputtext](https://www.primefaces.org/primeng/showcase/#/inputtext)